web/views/setting_view.ex

```diff
@@ -1,8 +1,17 @@
 defmodule CoursePlanner.SettingView do
   @moduledoc false
   use CoursePlanner.Web, :view
+  alias CoursePlanner.SharedView
 
-  def title do
-    "Settings"
+  def setting_input(form, field, type, label) do
+    case type do
```

Comment:
the same for me. I originally put them there in case we wanna show some different ways in future. but I can as well removed them for now